pyshelf/search/connection.py

```diff
@@ -55,9 +55,6 @@ def _parse_url(self):
         """
         self._parsed_url = urlparse(self.connection_string)
 
-        if not self._parsed_url.scheme:
```

Comment:
I see no reason to keep this since Elasticsearch-py defaults this for us. Also _parsed_url is internal so the scheme isn't referenced anywhere else. The only reason we care about it is if it is https we want to set ssl and verify certs to True.